09_iterators.md

```diff
@@ -190,16 +189,16 @@ But, with far and tiny pointers, you don't know what the reference
 type should be.
 So now we need to provide it.
 
-4. `reference`: the type of a reference to the value.
-5. `pointer`:  the type of a pointer to the value.
+&nbsp;&nbsp;&nbsp;4. `reference`: the type of a reference to the value.<br/>
+&nbsp;&nbsp;&nbsp;5. `pointer`:  the type of a pointer to the value.
```

Comment:
Does inserting an additional line break instead of `<br/>` work? I would like to avoid HTML if possible.

Reply:
It works, but looks a bit weird, because the spacing isn't the same as with a normal list. I think it's probably best if you try both ways and make a call.